superset/assets/javascripts/SqlLab/components/SqlEditor.jsx

```diff
@@ -148,7 +148,7 @@ class SqlEditor extends React.PureComponent {
         <div className="pull-left">
           <Form inline>
             <RunQueryActionButton
-              allowAsync={this.props.database && this.props.database.allow_run_async}
+              allowAsync={this.props.database ? this.props.database.allow_run_async : false}
```

Comment:
We use this in a fair amount of places. The casting to boolean is implicit. I know the equivalent is considered pythonesque in python but not sure about JS...